.github/workflows/ci.yml

```diff
@@ -28,14 +28,15 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+    - name: Set up JDK 11
```

Comment:
Why 11 and not 14 if you want to upgrade?

Reply:
11 is the one that ships with Android studio and the one needed to run the Google check style gradle plugin 😄